core/domain/visualization_registry.py

```diff
@@ -43,10 +43,8 @@ def _refresh_registry(cls):
 
             ancestor_names = [
                 base_class.__name__ for base_class in inspect.getmro(clazz)]
-            if 'BaseVisualization' not in ancestor_names:
```

Comment:
Shouldn't there be a replacement test for this, similar to value generators?

Reply:
I did not remove the test to check for BaseVisualization, I just modified the structure of the code such that it becomes testable.

Reply:
Hm, let me rephrase: where is the check, similar to what you have in ValueGeneratorNameTests, that ensures the correct set of visualizations is captured? Is there already such a check in the codebase? If not, we should add one.

Reply:
Done!